app/src/ui/diff/side-by-side-diff-row.tsx

```diff
@@ -138,8 +148,15 @@ export class SideBySideDiffRow extends React.Component<
     this.state = { showWhitespaceHint: undefined }
   }
   public render() {
-    const { row, showSideBySideDiff } = this.props
-
+    const {
+      row,
+      showSideBySideDiff,
+      beforeClassNames,
+      afterClassNames,
+    } = this.props
+
+    const beforeClasses = classNames('before', ...beforeClassNames)
+    const afterClasses = classNames('after', ...afterClassNames)
```

Comment:
I think we can just se the array here, no need to spread it
```suggestion
    const beforeClasses = classNames('before', beforeClassNames)
    const afterClasses = classNames('after', afterClassNames)
```

Reply:
This is because they're readonly.
![image](https://user-images.githubusercontent.com/75402236/156610264-83246b7c-17f7-49d4-b047-6cea4cba03b7.png)

Reply:
Ah, that's a bummer. Thanks for checking!